.github/workflows/nodejs.yml

```
@@ -160,14 +162,14 @@ jobs:
           toolchain: stable
           targets: wasm32-wasip1
       - name: Cache cargo
-        uses: actions/cache@v4
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)